contracts/wveOLAS.sol

```diff
@@ -111,21 +111,40 @@ error WrongTimestamp(uint256 minTimeStamp, uint256 providedTimeStamp);
 /// @param ve Original veOLAS address.
 error ImplementedIn(address ve);
 
+/// @dev veOLAS token is non-transferable.
+/// @param ve veOLAS token address.
+error NonTransferable(address ve);
+
+/// @dev veOLAS token is non-delegatable.
+/// @param ve veOLAS token address.
+error NonDelegatable(address ve);
+
 /// @title wveOLAS - Wrapper smart contract for view functions of veOLAS contract
 /// @author AL
 /// @author Aleksandr Kuperman - <aleksandr.kuperman@valory.xyz>
 contract wveOLAS {
     // veOLAS address
     address public immutable ve;
+    // OLAS address
+    address public immutable token;
+    // veOLAS token name
+    string public constant name = "Voting Escrow OLAS";
```

Comment:
so we use the same name, we do not use the word wrapper here

Reply:
no, as we need to return the veOLAS result at the end